python/lsst/cp/pipe/linearity.py

```diff
@@ -215,6 +220,42 @@ class LinearitySolveConfig(pipeBase.PipelineTaskConfig,
         doc="Maximum number of rejections per iteration for spline fit.",
         default=5,
     )
+    doSplineFitOffset = pexConfig.Field(
+        dtype=bool,
+        doc="Fit a scattered light offset in the spline fit.",
+        default=True,
+    )
+    doSplineFitWeights = pexConfig.Field(
+        dtype=bool,
+        doc="Fit linearity weight parameters in the spline fit.",
+        default=False,
+    )
+    splineFitWeightParsStart = pexConfig.ListField(
+        dtype=float,
+        doc="Starting parameters for weight fit, if doSplineFitWeights=True. "
+            "Parameters are such that sigma = sqrt(par[0]**2. + par[1]**2./mu).",
```

Comment:
I have questions about this, but after reading the rest of the changes (and the test comment), it seems like these aren't being fit as part of the standard pipeline code, so I think my questions aren't relevant at the moment.